src/lib/Theme.ts

```diff
@@ -53,6 +53,37 @@ export default class Theme {
     static SRC_FOLDER = './.fdk/temp-theme';
     static SRC_ARCHIVE_FOLDER = './.fdk/archive';
     static ZIP_FILE_NAME = `archive.zip`;
+    static SETTINGS_DATA_PATH = path.join(process.cwd(), '/theme/config/settings_data.json');
+    static SETTINGS_SCHEMA_PATH = path.join(process.cwd(), '/theme/config/settings_schema.json');
+
+    public static async writeSettingJson(path, options) {
+        try {
+            await fs.writeJSON(
+                path,
+                options,
```

Comment:
change options to jsonOject because it contains JSON object. @Shivraj0

Reply:
Done @vivek8690